R/rsqCalc.R

```diff
@@ -26,8 +25,8 @@ rsqCalc <- function(normMatrix, fit) {
                             msg = "fit must be of class 'MArrayLM'.")
     assertthat::assert_that(is.numeric(as.matrix(normMatrix)),
                             msg = "All of the entries in normMatrix must be numeric.")
-
-    sst <- Matrix::rowSums(normMatrix^2)
+    browser()
+    sst <- rowSums(normMatrix^2)
```

Comment:
where is rowSums coming from if not Matrix?  I think the base version is "rowsum"

Reply:
@cb4ds I found a base `rowSums()` function in addition to the `rowsum()` function: ![image](https://user-images.githubusercontent.com/64991568/101662930-1459fd00-3a18-11eb-8fb9-7788ff0472a8.png)

Reply:
excellent, thanks!